packages/shorebird_cli/lib/src/commands/release/release_android_archive_command.dart

```diff
@@ -8,6 +8,7 @@ import 'package:mason_logger/mason_logger.dart';
 import 'package:path/path.dart' as p;
```

Comment:
can we rename the file for consistency to `release_aar_command.dart`?

Reply:
Whoops, missed this one. Done!